README.Rmd

```diff
@@ -125,14 +125,14 @@ the package.
 
 <details> <summary> End-to-end workflows </summary>
 
-The [workflow vignette](articles/estimate_infections_workflow.html) provides guidance on the end-to-end process of estimating reproduction numbers and performing short-term forecasts for a disease spreading in a given setting.
+The [workflow vignette](./articles/estimate_infections_workflow.html) provides guidance on the end-to-end process of estimating reproduction numbers and performing short-term forecasts for a disease spreading in a given setting.
```

Comment:
I couldn't figure out why the original links are not working.

Reply:
and this fixed it?

Reply:
No, it didn't. I don't think it is easy to make the links in the README relative. I didn't find anything useful in the pkgdown documentation. I think we may have to use the absolute links of either the dev or stable version.

Reply:
Could we use `vignette` statements instead? This means we can't have link text so will need some minor re-writing but I'm looking at the `pkgdown` documentation as an example:

The end of this line in the README
https://github.com/r-lib/pkgdown/blob/676cf1dce39e4dc5aa1bcb671b1e14e992589a0a/README.Rmd#L23
gets rendered to a simple `vignette("pkgdown")` in the README but a link [`vignette("pkgdown")`](https://pkgdown.r-lib.org/articles/pkgdown.html) in pkgdown.

In the line highlighted above this could be something like:
```markdown
The workflow vignette `vignette("estimate_infections_workflow")` provides guidance on the end-to-end process of estimating reproduction numbers and performing short-term forecasts for a disease spreading in a given setting.
```
or similar.

Reply:
That should work. See changes here 84175eb6b1ea1050dfe1892a74bf2376c2e2a966. I've also reworded the part about the website to refer explicitly to the stable and dev sites here c743acc90a438808e59a339d15705ee28a72deca.